packages/account-v2/src/components/SignupWizard/index.tsx

```diff
@@ -1,41 +1,69 @@
-import React, { useEffect } from 'react';
+import React, { useCallback, useEffect } from 'react';
 import ReactModal from 'react-modal';
 import { StandaloneXmarkBoldIcon } from '@deriv/quill-icons';
-import { useSignupWizardContext } from '../../context/SignupWizardContext';
+import { ACTION_TYPES, useSignupWizardContext } from '../../context/SignupWizardContext';
+import { CUSTOM_STYLES } from '../../helpers/signupModalHelpers';
+import { stepProgress } from '../../mocks/form-progress.mock';
 import CurrencySelector from '../../pages/CurrencySelector';
-import { CUSTOM_STYLES } from './helpers';
+import { FormProgress } from '../form-progress';
 import './index.scss';
 
+/**
+ * @name WizardScreens
+ * @description The WizardScreens component is used to render the screens in the SignupWizard component based on the active step from `SignupWizardContext`.
+ * @example
+ * return (
+ *   <WizardScreens />
+ * );
+ */
 const WizardScreens = () => {
     const { currentStep } = useSignupWizardContext();
 
     // eslint-disable-next-line sonarjs/no-small-switch
     switch (currentStep) {
         case 1:
             return <CurrencySelector />;
+        case 2:
+            return <CurrencySelector />;
+        case 3:
+            return <CurrencySelector />;
+        case 4:
+            return <CurrencySelector />;
         default:
             return null;
     }
 };
 
+/**
+ * @name SignupWizard
+ * @description The SignupWizard component is used to render the signup wizard modal.
+ * @example
+ * return (
+ *  <SignupWizard />
+ * );
+ */
 const SignupWizard: React.FC = () => {
-    const { isWizardOpen, setIsWizardOpen } = useSignupWizardContext();
+    const { currentStep, dispatch, helpers, isWizardOpen, setIsWizardOpen } = useSignupWizardContext();
     useEffect(() => {
         ReactModal.setAppElement('#v2_modal_root');
     }, []);
+
+    const handleClose = useCallback(() => {
+        setIsWizardOpen(false);
+        dispatch({ payload: {}, type: ACTION_TYPES.RESET });
+        helpers.setStep(1);
```

Comment:
what does helpers mean? can we have a better name ?

Reply:
these are helper functions from the useStep hook, we can definitely rename it. Would you please suggest a suitable name?